### ISSUE TITLE
Bad Layout of Simple Bar Graph

### ISSUE THREAD

**Issue by leegee** [theiviaxx/MilkChart#25]
I'm going to try to attach a screenshot.

The chart rendering has the bars offset so that they do not sit on the axis, and over-lay on the zero axis.

``` html
<table id="histogram">
    <thead>
        <tr><th>V</th>
    </tr></thead>
    <tbody>
        <tr><td>1</td></tr>
        <tr><td>2</td></tr>
        <tr><td>3</td></tr>
        <tr><td>4</td></tr>
        <tr><td>5</td></tr>
        <tr><td>6</td></tr>
        <tr><td>7</td></tr>
        <tr><td>8</td></tr>
        <tr><td>10</td></tr>
    </tbody>
</table>
```

``` javascript
var chart = new window.MilkChart.Bar( $('histogram') );
```

![Screen shot 2013-02-15 at 11 35 57](https://f.cloud.github.com/assets/512816/160215/bd6098c4-775b-11e2-9ab8-1204823a6f63.png)

**Comment by theiviaxx** [comment]
Good catch, i'll get this fixed

**Comment by theiviaxx** [comment]
fixed in 1.5.9
